python/ray/worker.py

```diff
@@ -724,6 +725,30 @@ def array_custom_deserializer(serialized_obj):
         custom_serializer=array_custom_serializer,
         custom_deserializer=array_custom_deserializer)
 
+    # Define a custom serializer and deserializer for handling xarray DataArray.
```

Comment:
Let's actually do all of these in a big try/except block and also do the `import xarray` here in that block.